src/config/typeorm/index.ts

```diff
@@ -2,21 +2,21 @@ export default [
   {
 		"name": "default",
 		"type": "mysql",
-		"host": "localhost",
+		"host": process.env.DATASOURCE_HOST,
 		"port": 3306,
```

Comment:
Could potentially be environment variable as well in order to keep them in one place